tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -18,47 +18,47 @@
+    <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.405.22" />
 
     <!-- Integration: AWSSDK.EventBridge -->
     <!--    Assembly: AWSSDK.EventBridge -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.EventBridge/3.7.402.33 -->
-    <PackageReference Include="AWSSDK.EventBridge" Version="3.7.402.33" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.EventBridge/3.7.402.37 -->
+    <PackageReference Include="AWSSDK.EventBridge" Version="3.7.402.37" />
 
     <!-- Integration: AWSSDK.Kinesis -->
     <!--    Assembly: AWSSDK.Kinesis -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Kinesis/3.7.402.68 -->
-    <PackageReference Include="AWSSDK.Kinesis" Version="3.7.402.68" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Kinesis/3.7.402.72 -->
+    <PackageReference Include="AWSSDK.Kinesis" Version="3.7.402.72" />
 
     <!-- Integration: AWSSDK.S3 -->
     <!--    Assembly: AWSSDK.S3 -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.S3/3.7.414.1 -->
-    <PackageReference Include="AWSSDK.S3" Version="3.7.414.1" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.S3/3.7.415 -->
+    <PackageReference Include="AWSSDK.S3" Version="3.7.415" />
 
     <!-- Integration: AWSSDK.SimpleNotificationService -->
     <!--    Assembly: AWSSDK.SimpleNotificationService -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SimpleNotificationService/3.7.400.91 -->
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.400.91" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SimpleNotificationService/3.7.400.95 -->
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.400.95" />
 
     <!-- Integration: AWSSDK.SQS -->
     <!--    Assembly: AWSSDK.SQS -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SQS/3.7.400.91 -->
-    <PackageReference Include="AWSSDK.SQS" Version="3.7.400.91" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SQS/3.7.400.95 -->
+    <PackageReference Include="AWSSDK.SQS" Version="3.7.400.95" />
 
     <!-- Integration: AWSSDK.StepFunctions -->
     <!--    Assembly: AWSSDK.StepFunctions -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.StepFunctions/3.7.403.32 -->
-    <PackageReference Include="AWSSDK.StepFunctions" Version="3.7.403.32" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.StepFunctions/3.7.403.38 -->
+    <PackageReference Include="AWSSDK.StepFunctions" Version="3.7.403.38" />
 
     <!-- Integration: Azure.Messaging.ServiceBus -->
     <!--    Assembly: Azure.Messaging.ServiceBus -->
-    <!-- Latest package https://www.nuget.org/packages/Azure.Messaging.ServiceBus/7.18.3 -->
+    <!-- Latest package https://www.nuget.org/packages/Azure.Messaging.ServiceBus/7.18.4 -->
     <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.17.5" />
```

Comment:
Needs manual testing

Reply:
This one and the CosmosDB one

I'll toss both of them on my plate to test